src/Illuminate/Events/Dispatcher.php

```diff
@@ -268,6 +268,27 @@ public function dispatch($event, $payload = [], $halt = false)
         return $halt ? null : $responses;
     }
 
+    /**
+     * Dispatch multiple events and call their listeners.
+     *
+     * @param  array<int|string, mixed>  $events
+     * @return void
+     */
+    public function dispatchMany($events)
```

Comment:
What about halting and getting response? See dispatch() signature.

Reply:
Yeah — I didn't know if it made a lot of sense to get a response from dispatching multiple events. Maybe return an array if `$halt = true`?